package.yaml

```diff
@@ -33,6 +33,7 @@ dependencies:
 - wai-extra >= 3.0 && < 4
 - vault >= 0.3 && < 1
 - cryptonite < 1
+- cryptostore >= 0.2 && <0.3
```

Comment:
I was surprised that `x509-store` doesn't seem to provide the required functionality here and an extra dependency is needed.

Reply:
I actually don't remember why exactly this was needed... Let me check again